Sources/DVR/SessionDataTask.swift

```diff
@@ -11,7 +11,7 @@ final class SessionDataTask: URLSessionDataTask {
 
     // MARK: - Properties
 
-    weak var session: Session!
+    var session: Session!
```

Comment:
Why the removal of `weak` here?

Reply:
So long story to be made short, I ran into an interesting error where depending on how long your hook was, the session would de-initialize itself before the code finished running. I did a lot of research on how things are reinitialized and tracked it down to this assignment. To keep session from getting de-initialized before the completion of the code, it needed a strong reference.